### ISSUE TITLE
confused about how to $state.transitionTo sibling state

### ISSUE THREAD

**Issue by btm1** [angular-ui/ui-router#365]
so i have a few states:

```
    .state('share.contacts', {
        abstract: true,
        url: "/contacts",
        templateUrl: "/inc/share/contacts/index.html"
    })
        .state('share.contacts.manage', {
          abstract: true,
          url: "/manage/:page",
          templateUrl: "/inc/share/contacts/manage.html",
          controller: "contactsController"
        })
            .state('share.contacts.manage.contact', {
              url: "/:contact",
              templateUrl: "/inc/share/contacts/detail.html",
              controller: "contactDetail"
                .state('share.contacts.manage.contact.info', {
                    url: "/info",
                    templateUrl: "/inc/share/contacts/info.html",
                })
                .state('share.contacts.manage.contact.history', {
                    url: "/history",
                    templateUrl: "/inc/share/contacts/history.html",
                })
                .state('share.contacts.manage.contact.notes', {
                    url: "/notes",
                    templateUrl: "/inc/share/contacts/notes.html",
                })
```

These produce partially dynamic urls i.e.:
/share/contacts/manage/1/39095/info

I have a button on the page and i want to tell the state to .transitionTo
/share/contacts/manage/1/39095/history
how do i produce the $state.transitionTo that inherits the previous url and just swaps the end?

I've tried: $state.transitionTo('share.contacts.manage.contact.history','relative')

basically i want to say transition to specific state x and inherit the current url params or i want to say transition to sibling state x and inherit url params
